src/Calendar.php

```diff
@@ -110,6 +110,15 @@ public function addEvent(EventInterface $event)
         return $this;
     }
 
+    public function addEvents(array $events)
```

Comment:
Could you add a docblock? Especially mentioning that non EventInterface objects will be ignored.